import java.io.IOException;

/**
 * 测试主类
 */

public class TestMain {
public static void main(String[] args) throws Exception{



    TCP.startTCP();

	//启动SocketClient

	//TestStop.task.cancel();
    //TestStop.timer.cancel();
	/*ConnectThread connectSocket = new ConnectThread();
	System.out.println("Starting server...running in background");
	//启动检测心跳包线程
	//connectSocket.start();
	//启动服务端监听
	StartSocket.StratListening(1027);
*/

	//废弃代码
	//启动SocketServer
	/*System.out.println("Starting client...");
	try {
		SocketClient.StartClient();
	} catch (IOException e) {
		// TODO Auto-generated catch block
		e.printStackTrace();
	}*/
	
	

}
}
